scripts/build/build_node/Platform/Windows/install-windows-android.ps1

```diff
@@ -29,15 +23,28 @@ Start-Process -FilePath $sdkmanager -ArgumentList $build_tools -NoNewWindow -Wai
 Start-Process -FilePath $sdkmanager -ArgumentList $ndk -NoNewWindow -Wait
 
 Write-Host "Installing Gradle"
-Import-Module C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1 #Grade needs a custom installer due to being hardcoded to C:\Programdata in Chocolatey
+$gradle_version = '7.0'
+$gradle_checksum = '81003F83B0056D20EEDF48CDDD4F52A9813163D4BA185BCF8ABD34B8EEEA4CBD'
+
+#Gradle needs a custom installer due to being hardcoded to C:\Programdata in Chocolatey
+Import-Module C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1 
 $packageName = 'gradle'
 $url = "https://services.gradle.org/distributions/gradle-$gradle_version-all.zip"
 $installDir = "C:\Gradle"
 
 Install-ChocolateyZipPackage $packageName $url $installDir -Checksum $gradle_checksum -ChecksumType 'sha256'
 
-$gradle_home = Join-Path $installDir "$packageName-$version"
+$gradle_home = Join-Path $installDir "$packageName-$gradle_version"
 [Environment]::SetEnvironmentVariable("GRADLE_BUILD_HOME", $gradle_home, [EnvironmentVariableTarget]::Machine)
+[Environment]::SetEnvironmentVariable(
+    "Path",
+    [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";$gradle_home\bin",
```

Comment:
Shouldn't "path" be all caps?
```suggestion
    "PATH",
    [Environment]::GetEnvironmentVariable("PATH", [EnvironmentVariableTarget]::Machine) + ";$gradle_home\bin",
```

Reply:
It's case insensitive for Windows, but I can change this for consistency